nixos/modules/services/monitoring/prometheus/exporters/shelly.nix

```diff
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.shelly;
+in
+{
+  extraOpts = {
+    listen-address = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      example = "127.0.0.1";
+      description = lib.mdDoc ''
+        The address to listen on for HTTP requests
+      '';
+    };
+    listen-port = mkOption {
+      type = types.port;
+      default = 9784;
+      example = 9784;
```

Comment:
```suggestion
```
If example is the same as default we should omit it.

Reply:
Thanks, just fixed it.